File: Python/counter.py
def counter(quarters, dimes, nickels, pennies):
    total = 25*quarters + 10*dimes + 5*nickels + pennies
    dollars, cents = total//100, total % 100
    return dollars, cents


if __name__ == "__main__":
    print("Please enter the number of coins: ")
    q = int(input("# of quarters: "))
    d = int(input("# of dimes: "))
    n = int(input("# of nickels: "))
    p = int(input("# of pennies: "))
    d, c = counter(q, d, n, p)
    if d == 1:
        print(f"The total is {d} dollar and {c} cents")
    else:
        print(f"The total is {d} dollars and {c} cents")

File: C++/counter.cpp
#include <iostream>


int main() {
    int quarters, dimes, nickels, pennies;
    int total, dollars, cents;

    std::cout << "Please enter the number of coins:" << std::endl;

    std::cout << "# of quarters: ";
    std::cin >> quarters;

    std::cout << "# of dimes: ";
    std::cin >> dimes;

    std::cout << "# of nickels: ";
    std::cin >> nickels;

    std::cout << "# of pennies: ";
    std::cin >> pennies;

    total = 25*quarters + 10*dimes + 5*nickels + pennies;
    dollars = total / 100;
    cents = total % 100;
    std::cout << "The total is " << dollars << " dollars and " << cents << " cents";

    return 0;
}
